data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,1183 @@
+		government "Heroic Unfettered"
+		personality heroic escort disables plunders
+		ship "Flagship Escort" "Captain Oooopeep"
+		ship "Flagship Escort" "Captain Nerpi~i"
+	npc
+		government "Korath"
+		system Alcyone
+		personality disables plunders opportunistic harvests staying
+		fleet "Korath Raid" 3
+	npc kill
+		government "Pirate"
+		system Alcyone
+		personality disables plunders opportunistic harvests staying target
+		fleet "Hired Guns" 1
+		fleet "Large Core Pirates" 3
+		fleet "Small Core Pirates" 8
+
+
+mission "Hai Slave Rescue [14]"
+	name "Hai Slave Rescue: <planet>"
+	description "Meet the Centipedes at <destination>. Payment is <payment>."
+	landing
+	source Stormhold
+	destination Canyon
+	clearance
+	passengers 1
+	blocked `You cannot save the Hai Slaves unless you have one bunk for Teeneep.`
+	to offer
+		has "Hai Slave Rescue [13]: done"
+	on offer
+		log `Conquered Stormhold. The Unfettered Hai are pillaging it for, "reparations," while I retrieve the nine Centipedes.`
+		conversation
+			`As you land on Stormhold, you see a disturbing sight: twenty-five Unfettered Hai Geocoris ships jump into the system and land next to you. They've been modified as troop transports; hundreds of Unfettered troops step out alongside massive battle tanks. The troops quickly overcome defenders, leveling buildings and destroying underground bunkers. The Unfettered shoot with extreme precision, destroying some rooms of buildings and leaving nearby rooms untouched. These are terrifyingly effective troops.`
+			# You see now what millions of Wanderers have seen on their own planets.
+			
+			`	Of the 913 Hai slaves, 871 were rescued. The others had died in their own escape attempt a week earlier. They managed to detonate a generator and destroy half the facility before their deaths though, which made the Unfettered rescue a little easier.`
+			# Did Navy or Syndicate agents have something to do with that escape attempt
+			# and sabotage? Has Teeneep just turned a few agents into red paste and
+			# scorch marks?
+			
+			`	Teeneep approaches you, "We've found all the surviving Hai, including my family. Now you need to take me to <planet>, so we can retrieve the Hai Centipedes. The True Hai will stay here collecting restitution from Stormhold while we're away. You'll receive a share once all Hai are safely home. It will be a large payment."`
+				accept
+			# "Collecting restitution payments." The Unfettered Hai are pillaging
+			# a planet of pillagers.
+	on complete
+		payment 75000
+
+
+mission "Hai Slave Rescue [15]"
+	name "Hai Slave Rescue: <planet>"
+	description "Escort the Centipedes to <destination>. Payment is <payment>."
+	landing
+	destination Stormhold
+	source Canyon
+	clearance
+	passengers 1
+	to offer
+		has "Hai Slave Rescue [14]: done"
+	on offer
+		conversation
+			`The Hai prepare their nine centipedes for the trip to <planet>. Hopefully there won't be a pirate ambush waiting; you have no Unfettered helping you this time.`
```

Comment:
Why on earth would the unfettered NOT want to make sure the transports are safe?
```suggestion
			`The Hai prepare their nine centipedes for the trip to <planet>. Hopefully the pirates won't mount a counterattack.`
```